index.html

```diff
@@ -0,0 +1,311 @@
+    />
+    <meta
+      name="keywords"
+      content="scholarly, publishing, publisher, manuscript, document, academic, education, open, collaborative, reproducible, free, edit, collaborate, ideas, version control, code, writing, automatic, digital, interactive, print, fair, hackable"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Quicksand"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Inconsolata"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
+    <link rel="manifest" href="/site.webmanifest" />
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#ad1457" />
+    <meta name="msapplication-TileColor" content="#ad1457" />
+    <meta name="theme-color" content="#ad1457" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <div class="logo_container">
+        <img class="logo logo_shadow" src="logo-animated.svg" />
+        <img class="logo" src="logo-animated.svg" />
+      </div>
+      <div class="logo_text">
+        Manubot
+      </div>
+      <div class="logo_subtext">
+        Publishing, open and automated.
+        <span class="cursor_blink">|</span>
+      </div>
+    </header>
+    <section class="special_section">
+      <div class="row flex_wrap">
+        <a href="" class="button_link">
+          <i class="fas fa-graduation-cap"></i>
+          Get Started
+        </a>
+        <a href="" class="button_link">
+          <i class="fas fa-lg fa-book"></i>
+          Docs
+        </a>
+        <a href="" class="button_link">
+          <i class="fab fa-lg fa-github"></i>
+          On GitHub
+        </a>
+      </div>
+    </section>
+    <section>
+      <div class="row flex_wrap">
+        <div class="col_image"></div>
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-asterisk fa-lg"></i> At a glance
+          </div>
+          <div>
+            Manubot is a workflow and set of tools for the next generation of scholarly publishing.
+            Write your manuscript in <a href="https://www.markdownguide.org/">markdown</a>,
+            track it with <a href="https://git-scm.com/">git</a>,
+            automatically convert it to <code>.html</code>, <code>.pdf</code>, or <code>.docx</code>,
+            and deploy it to your <a href="https://pages.github.com/">destination</a> of choice.
```

Comment:
https://github.com/manubot/manubot.org/pull/6#discussion_r286101044
> Since this sentence is so long, it is an occasion where you could do a newline after each phrase.